src/uwtools/api/driver.py

```diff
@@ -0,0 +1,28 @@
+"""
+API access to the ``uwtools`` driver base classes.
+"""
+
+import importlib
+import sys
+
+_CLASSNAMES = [
+    "Assets",
+    "AssetsCycleBased",
+    "AssetsCycleAndLeadtimeBased",
+    "AssetsTimeInvariant",
+    "Driver",
+    "DriverCycleBased",
+    "DriverCycleAndLeadtimeBased",
+    "DriverTimeInvariant",
+]
```

Comment:
It's pretty nifty to have a place to see what all our drivers' names are. I know there are other ways to get the info, but this is nice.